packages/ui/src/components/va-slider/VaSlider.vue

```diff
@@ -515,9 +515,21 @@ export default defineComponent({
       }
     }
 
+    const pinPositionStep = computed(() => {
+      if (props.min === 0 && props.max === 100 && props.step !== 1) {
+        // only step prop passed
+        return props.step
+      } else if ((props.min !== 0 || props.max !== 100) && props.step === 1) {
+        // only min or/and max prop passed
+        return (100 / (props.max - props.min))
+      } else {
+        // step and (min or max) prop passed
+        return ((props.step) * 100) / (props.max - props.min)
```

Comment:
Can't we just use this last formula?

Reply:
Yes, you're right. There was another formula before, but now (taking into account, that step can be === 1 in the second case), they're the same.